src/ntlm/mod.rs

```diff
@@ -368,6 +368,50 @@ impl Ntlm {
             expiry: None,
         })
     }
+
+    fn compute_checksum(
+        &mut self,
+        message: &mut [SecurityBuffer],
+        sequence_number: u32,
+        digest: &[u8; 16],
+    ) -> crate::Result<SecurityStatus> {
+        let checksum = self
+            .send_sealing_key
+            .as_mut()
+            .unwrap()
+            .process(&digest[0..SIGNATURE_CHECKSUM_SIZE]);
+
+        let signature_buffer = SecurityBuffer::find_buffer_mut(message, SecurityBufferType::Token)?;
+        if signature_buffer.buf_len() < SIGNATURE_SIZE {
+            return Err(Error::new(ErrorKind::BufferTooSmall, "The Token buffer is too small"));
```

Comment:
```suggestion
            return Err(Error::new(ErrorKind::BufferTooSmall, "the Token buffer is too small"));
```

I know you moved this code from the `encrypt_message` function. It was written a long time ago and we are now following the new style guide (it is undocumented, but you can use this one as a reference: https://github.com/Devolutions/IronRDP/blob/master/STYLE.md#formatting-of-error-messages).